src/hooks/useEmail.tsx

```diff
@@ -113,14 +113,15 @@ export function useEmail(): IEmailSender {
     }
 
     const sendAcceptedEmail = async (request: IRequirementsRequestCRUD, comment?: string): Promise<void> => {
-        let to = [request.Requester, request.Approver, request.Author];
-        let subject = `Request ${request.getFormattedId()} Accepted by Manager`;
-        let body = `Hello, Requirement Request ${request.getFormattedId()}, ${request.Title}, for ${request.ApplicationNeeded !== ApplicationTypes.OTHER ? request.ApplicationNeeded : request.OtherApplicationNeeded} has been approved/accepted by the Requirements Manager ${(await userApi.getCurrentUser()).Title}. The next step for the Request is for it to be reviewed by the CITO. 
+        let to = getUsersWithRole(RoleType.CITO);
+        let cc = [request.Requester, request.Approver, request.Author];
+        let subject = `Request ${request.getFormattedId()} Needs CITO Review`;
+        let body = `Hello, Requirement Request ${request.getFormattedId()}, ${request.Title}, for ${request.ApplicationNeeded !== ApplicationTypes.OTHER ? request.ApplicationNeeded : request.OtherApplicationNeeded} has been approved/accepted by the Requirements Manager ${(await userApi.getCurrentUser()).Title} and now the Request needs to be reviewed by the you, as the CITO. 
         ${comment ? `The Manager left a comment saying "${comment}"` : ''}
         
         To view the request and any comments/modifications left by the manager, please copy the following link and paste it in your browser ${emailApi.siteUrl}/app/index.aspx?route=%23%2FRequests%2FView%2F${request.Id}`;
```

Comment:
Since this message CC's some folks, I think the body should be a little more generic.

> ...has been approved/accepted by the Requirements Manager, [manager]. The Request is now ready for CITO review.
> 
> To (re)view the request...

Thoughts?

Reply:
I like that, I wasn't really sure about what to say but I think that's good